core/src/services/cloudService.ts

```diff
@@ -1,4 +1,25 @@
-import { StringParams } from "../common/stringParams";
+import {  StringParams } from "../common/stringParams";
+
+/**
+ * Invoke Request options
+ * Implemented in order to have all the parameters needed for the invoke function in one object
+ */
+export interface InvokeRequest {
+  /**
+   * @param name Function name to invoke
+   * @param fireAndForget Wait for response if false (default behavior)
+   * @param payload HTTP request body
+   * @param headers Context headers
+   * @param pathParams StringParams with values for URL
+   * @param extraParams Extra paramters to invoke function
```

Comment:
nit: Extra parameters

Reply:
Thanks for this feedback, It will be apply in my next commit